AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.ts

```diff
@@ -30,7 +30,6 @@ export class SearchEmailComponent implements OnInit, OnDestroy {
     isJoh = false;
     notFoundEmailEvent = new Subject<boolean>();
     notFoundEmailEvent$ = this.notFoundEmailEvent.asObservable();
-    searchPending = new BehaviorSubject<boolean>(false);
```

Comment:
Removed after search application if field is already blurred. Behaves same as participants